pkg/eventmanager/eventmanager.go

```diff
@@ -127,6 +140,12 @@ func (s *eventManager) Notify(state notifier.MuoState) error {
 		description = createFailureDescription(uc)
 	case notifier.MuoStateHealthCheck:
 		description = fmt.Sprintf(UPGRADE_HEALTHCHECK_DELAY_DESC, uc.Spec.Desired.Version)
+	case notifier.MuoStateControlPlaneUpgradeStartedSL:
+		description = fmt.Sprintf(UPGRADE_CONTROL_PLANE_STARTED_DESC, uc.Spec.Desired.Version)
+	case notifier.MuoStateControlPlaneUpgradeFinishedSL:
+		description = fmt.Sprintf(UPGRADE_CONTROL_PLANE_FINISHED_DESC, uc.Spec.Desired.Version)
+	case notifier.MuoStateWorkerPlaneUpgradeFinishedSL:
```

Comment:
I'm OK with the current implementation here. I just have one question that since we have control plane upgrade started notification, will customer complain that they didn't see worker plane upgrade start notification?

Reply:
Yeah, agreed we need to make it more clear. The SL was already fixed but I'll add another quick fix on the vars side as well.